client/src/app-components/ticket-query-list.js

```diff
@@ -65,9 +67,21 @@ class TicketQueryList extends React.Component {
             orderBy: filters.orderBy ? JSON.parse(filters.orderBy) : filters.orderBy,
             showOrderArrows: true,
             onChangeOrderBy: onChangeOrderBy,
+            onPageSizeChange: this.test.bind(this),
+            showDropDown: false
         };
     }
 
+    test(selectedIndex) {
+        const currentPath = window.location.pathname;
+        const urlQuery = searchTicketsUtils.getFiltersForURL({
+            filters: {pageSizeIndex: 2},
+            shouldRemoveCustomParam: true,
+            shouldRemoveUseInitialValuesParam: true
+        });
+        urlQuery && history.push(`${currentPath}${urlQuery}`);
+    }
```

Comment:
Write a better name for this function based on what it does, it seems you wrote it while developing, is it current? Is it used?